### PR TITLE
Bp 479 logic for create event type

### PR DESCRIPTION
**Pull request by julianweise**

### CODE REVIEW

File: src/CreateEventTypeView/CreateEventTypeView.js [hunk 1]
```diff
@@ -5,103 +5,185 @@ import {Container} from "react-grid-system";
...
+							primary={true}/>
+					</div>
+				</Container>
+			</div>
+		);
+	}
 }
 
-export default CreateEventTypeView;
+export default ConnectionComponent.argosConnector()(props => ({
```
Comment (hpibptlab) [inline src/CreateEventTypeView/CreateEventTypeView.js]: MAJOR Remove the unused function parameter "props". [rule](http://bpt-lab.org/sonarqube/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)

**Comment by hpibptlab** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'

File: src/CreateEventTypeView/CreateEventTypeView.js [hunk 2]
```diff
@@ -5,103 +5,185 @@ import {Container} from "react-grid-system";
...
+				eventTypeTimestampAttributeErrorText: ''
+			});
+		}
+	}
 
-        updatedAttributes[currentAttributeIndex].value = event.target.value;
+	submitForm() {
+		const errorState = {};
+		if(!this.state.eventTypeName) {
+			errorState['eventTypeNameErrorText'] = 'Please fill out this field';
```
Comment (MaximilianV) [inline src/CreateEventTypeView/CreateEventTypeView.js]: Maybe we extract it into a config variable, like _formFieldRequired: Please fill out ..._?

File: src/CreateEventTypeView/CreateEventTypeView.js [hunk 3]
```diff
@@ -5,103 +5,185 @@ import {Container} from "react-grid-system";
...
+				</Container>
+			</div>
+		);
+	}
 }
 
-export default CreateEventTypeView;
+export default ConnectionComponent.argosConnector()(props => ({
+	createEventType: state => ({
+		deleteQueryResponse: {
```
Comment (MaximilianV) [inline src/CreateEventTypeView/CreateEventTypeView.js]: How does `deleteQueryResponse` fits to `createEventType`?

File: src/CreateEventTypeView/CreateEventTypeView.js [hunk 4]
```diff
@@ -5,103 +5,185 @@ import {Container} from "react-grid-system";
...
 
-    onInputChange(event) {
-        const currentAttributeId = event.target.id.toString();
-        const updatedAttributes = this.state.attributes;
-        const attributeIds = updatedAttributes.map(attribute => {
-            return attribute.id;
-        });
-        const currentAttributeIndex = attributeIds.indexOf(currentAttributeId);
+	handleChangeEventTypeName(event) {
+		if(event.target.value) {
```
Comment (MaximilianV) [inline src/CreateEventTypeView/CreateEventTypeView.js]: Prevents the deletion of last character:

1. Type something in input field
1. Try to delete everything by pressing `back` multiple times, or marking all and then `back`
1. You will see that at least the last character is quite rebellious

This is slightly inconvenient for the user.

File: src/CreateEventTypeView/CreateEventTypeView.js [hunk 5]
```diff
@@ -5,103 +5,185 @@ import {Container} from "react-grid-system";
...
+		}
+	}
 
-        if (!event.target.value) {
-            updatedAttributes.splice(currentAttributeIndex, 1);
-            this.setState({attributes: updatedAttributes});
-            return;
-        }
+	handleChangeEventTypeTimestampAttribute(event) {
+		if(event.target.value) {
```
Comment (MaximilianV) [inline src/CreateEventTypeView/CreateEventTypeView.js]: Same as in `handleChangeEventTypeName`.

**Review by MaximilianV — CHANGES_REQUESTED** [review]

**Review by MaximilianV — APPROVED** [review]

**Pull request merged**